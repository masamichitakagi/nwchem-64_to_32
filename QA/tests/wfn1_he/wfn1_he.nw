echo
start wfn1_he_dat

geometry
  he 0 0 0
end
basis
  * library 6-31g*
end

set wfn1:print_error T
set wfn1:maxit 20
set wfn1:t_bath 0.0

task wfn1 energy

set wfn1:print_error T
set wfn1:maxit 50
set wfn1:t_bath 0.5

task wfn1 energy
