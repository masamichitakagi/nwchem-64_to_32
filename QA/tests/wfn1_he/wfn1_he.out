 argument  1 = /Users/vandam/nwchem-wfn1-dev/QA/../QA/tests/wfn1_he/wfn1_he.nw



============================== echo of input deck ==============================
echo
start wfn1_he_dat

geometry
  he 0 0 0
end
basis
  * library 6-31g*
end

set wfn1:print_error T
set wfn1:maxit 20
set wfn1:t_bath 0.0

task wfn1 energy

set wfn1:print_error T
set wfn1:maxit 50
set wfn1:t_bath 0.5

task wfn1 energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.6
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = 10-4-27-16.user.vpn.bnl.local
    program         = /Users/vandam/nwchem-wfn1-dev/QA/../bin/MACX64/nwchem
    date            = Fri May 26 09:27:56 2017

    compiled        = Fri_May_26_09:27:12_2017
    source          = /Users/vandam/nwchem-wfn1-dev
    nwchem branch   = Development
    nwchem revision = 28658
    ga revision     = 10724
    input           = /Users/vandam/nwchem-wfn1-dev/QA/../QA/tests/wfn1_he/wfn1_he.nw
    prefix          = wfn1_he_dat.
    data base       = /Users/vandam/nwchem-wfn1-dev/QA/scratchdir/wfn1_he_dat.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = /Users/vandam/nwchem-wfn1-dev/QA/scratchdir
  0 scratch   = /Users/vandam/nwchem-wfn1-dev/QA/scratchdir




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 he                   2.0000     0.00000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      he                 4.002600


 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     1
 geometry
 he                    0.00000000     0.00000000     0.00000000

  library name resolved from: environment
  library file name is: </Users/vandam/nwchem-wfn1-dev/QA/../src/basis/libraries/>
  


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           6-31g*                   on all atoms 



                                NWChem WFN1 Module
                                ------------------


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  he (Helium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.84216340E+01  0.023766
  1 S  5.77803000E+00  0.154679
  1 S  1.24177400E+00  0.469630

  2 S  2.97964000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                          6-31g*                  2        2   2s



      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -2.85516040

      Non-variational initial energy
      ------------------------------

 Total energy =      -2.855160
 1-e energy   =      -3.882067
 2-e energy   =       1.026907
 HOMO         =      -0.914127
 LUMO         =       1.399859

 iteration              energy           error          err Na          err Nb          err Ca          err Cb
 ---------              ------           -----          ------          ------          ------          ------
         1         -2.85516043        0.000000        0.000000        0.000000        0.000000        0.000000

 Reached Convergence

 WFN1                     :
 --------------------------
 Total WFN1 energy =          -2.8551604262
 1-Electron energy =          -3.8820676078
 2-Electron energy =           1.0269071817
 Entropy energy    =           0.0000000000
 Nuclear energy    =           0.0000000000

 conventional Hartree-Fock:
 --------------------------
 Total SCF energy  =          -2.8551604262
 2-Electron energy =           1.0269071817


      Natural orbital occupation numbers

          Alpha     Beta
         ------   ------
     1   1.0000   1.0000
     2   0.0000   0.0000

                     WFN1 Final Alpha Natural Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=1.000000D+00  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -0.592081  1 He s                  2     -0.513586  1 He s          

 Vector    2  Occ=2.393441D-21  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.186959  1 He s                  1     -1.149818  1 He s          


                     WFN1 Final Beta Natural Orbital Analysis
                     ----------------------------------------

 Vector    1  Occ=1.000000D+00  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -0.592081  1 He s                  2     -0.513586  1 He s          

 Vector    2  Occ=2.393441D-21  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.186959  1 He s                  1     -1.149818  1 He s          


 Task  times  cpu:        0.1s     wall:        0.1s


                                NWChem Input Module
                                -------------------



                                NWChem WFN1 Module
                                ------------------



      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -2.85516040

      Non-variational initial energy
      ------------------------------

 Total energy =      -2.855160
 1-e energy   =      -3.882067
 2-e energy   =       1.026907
 HOMO         =      -0.914127
 LUMO         =       1.399859

 iteration              energy           error          err Na          err Nb          err Ca          err Cb
 ---------              ------           -----          ------          ------          ------          ------
         1         -2.85768599        0.026002        0.000000        0.000000        0.013001        0.013001
         2         -2.92541803        0.203829        0.000000        0.000000        0.101914        0.101914
         3         -3.16607687        0.000157        0.000015        0.000015        0.000064        0.000064
         4         -3.16615587        0.000146        0.000014        0.000014        0.000059        0.000059
         5         -3.16622931        0.000136        0.000014        0.000014        0.000054        0.000054
         6         -3.16629774        0.000127        0.000013        0.000013        0.000050        0.000050
         7         -3.16636166        0.000119        0.000013        0.000013        0.000047        0.000047
         8         -3.16642150        0.000111        0.000012        0.000012        0.000044        0.000044
         9         -3.16647762        0.000105        0.000012        0.000012        0.000041        0.000041
        10         -3.16653038        0.000099        0.000011        0.000011        0.000038        0.000038

 Reached Convergence

 WFN1                     :
 --------------------------
 Total WFN1 energy =          -3.1665303773
 1-Electron energy =          -3.5170510450
 2-Electron energy =           0.9949653577
 Entropy energy    =          -0.6444446900
 Nuclear energy    =           0.0000000000

 conventional Hartree-Fock:
 --------------------------
 Total SCF energy  =          -3.0543581856
 2-Electron energy =           1.1071375494


      Natural orbital occupation numbers

          Alpha     Beta
         ------   ------
     1   0.9013   0.9013
     2   0.0987   0.0987

                     WFN1 Final Alpha Natural Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=9.012977D-01  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -0.592802  1 He s                  2     -0.512842  1 He s          

 Vector    2  Occ=9.870235D-02  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.187280  1 He s                  1     -1.149447  1 He s          


                     WFN1 Final Beta Natural Orbital Analysis
                     ----------------------------------------

 Vector    1  Occ=9.012977D-01  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -0.592802  1 He s                  2     -0.512842  1 He s          

 Vector    2  Occ=9.870235D-02  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.187280  1 He s                  1     -1.149447  1 He s          


 Task  times  cpu:        0.1s     wall:        0.1s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:   88       88     5138      598     1932        0        0        8     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             1.36e+05 1.75e+04 6.12e+04 0.00e+00 0.00e+00 6.40e+01
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 1056 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        18	        17
	current total bytes		         0	         0
	maximum total bytes		   3203336	  22508648
	maximum total K-bytes		      3204	     22509
	maximum total M-bytes		         4	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
       S. A. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, V. Konjkov, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson,
      R. J. Harrison, J. R. Rehr, M. Dupuis, D. Silverstein, D. M. A. Smith,
            J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.2s     wall:        0.2s
